README.md

```diff
@@ -72,7 +72,10 @@ rustls does not and will not support:
 * TLS 1.2 protocol compression.
 * Discrete-log Diffie-Hellman.
 * Automatic protocol version downgrade.
-* Self-signed certificates.
+* Using CA certificates directly to authenticate a server/client (often called "self-signed
+certificates"). _CA certificates should only issue other certificates, and cannot be used
```

Comment:
"cannot be used alone for" seems a little awkward? This is a good direction, though!

Reply:
How about: "CA certificates should only issue other certificates, and are inappropriate for authentication in isolation." ? Less awkward?

Reply:
I don't really grok what you mean by "are inappropriate for authentication in isolation". I guess the real point is that authenticating a server against a certificate signed by its own private key, that is also marked as a CA, would actually be an okay way to authenticate, but we don't want to bear the extra complexity in webpki for enabling path building (safely) scenarios where this is the case?

In which case maybe we should say: "To limit complexity and risk in our path building implementation, our default verifier implementations do not support using an end-entity certificate as a trust anchor."?

Reply:
> In which case maybe we should say: "To limit complexity and risk in our path building implementation, our default verifier implementations do not support using an end-entity certificate as a trust anchor."?

Added text closer to this. WDYT?